src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -18,6 +18,8 @@
 @Component
 public class DataLoader
 {
+	
+	private static List<Map> placeCategories;
 
 	@PostConstruct
 	public void loadSomeData()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)